README.md

```diff
@@ -1,94 +1,142 @@
-
-Once ready for primetime, don't forget to revisit this `README.md` and update/replace it with details on your mixin.
-
-## Project Structure
-
-In the `cmd/docker` directory, you will find a cli built using [spf13/cobra](https://github.com/spf13/cobra). The CLI contains a go file for each basic capability a Mixin should implement:
-
-* build
-* schema
-* version
-* install
-* upgrade
-* invoke
-* uninstall
-
-Each of these command implementations have a corresponding Mixin implementation in the `pkg/docker` directory. Each of the commands above is wired into an empty implementation in `pkg/docker` that needs to be completed. In order to build a new Mixin, you need to complete these implementations with the relevant technology. For example, to build a [Cloud Formation](https://aws.amazon.com/cloudformation/) mixin, you might implement the methods in `pkg/docker` using the [AWS Go SDK](https://docs.aws.amazon.com/sdk-for-go/api/service/cloudformation/).
-
-## Provided capabilities
-
-This skeleton mixin project brings some free capabilities:
-
-### File System Access and Context
-
-Porter provides a [Context](https://github.com/deislabs/porter/tree/main/pkg/context) package that has helpful mechanisms for accessing the File System using [spf13/afero](https://github.com/spf13/afero). This makes it easy to provide mock File System implementations during testing. The Context package also provides a mechanism to encapsualte stdin, stdout and stderr so that they can easily be passed from `cmd/docker` code to implementing `pkg/docker` code.  
-
-### Template and Static Asset Handling
-
-The project already includes [Packr V2](https://github.com/gobuffalo/packr/tree/master/v2) for dealing with static files, such as templates or other content that is best modeled outside of a Go file. You can see an example of this in `pkg/docker/schema.go`.
-
-### Basic Schema
-
-The project provides an implementation of the `docker schema` command that is mostly functional. To fully implement this for your mixin, you simply need to provide a valid JSON schema. For reference, consult `pkg/docker/schema/schema.json`.
-
-### Basic Tests
-
-The project provides some very basic test skeletons that you can use as a starting point for building tests for your mixin.
-
-### Makefile
-
-The project also includes a Makefile that will can be used to both build and install the mixin. The Makefile also includes a TODO `publish` target that shows how you might publish the mixin and generate an mixin feed for easily sharing your mixin.
\ No newline at end of file
+# A Docker Mixin for Porter
+
+This is a Docker mixin for Porter. The mixin provides the Docker CLI. 
+
+## Mixin Declaration
+
+To use this mixin in a bundle, declare it like so:
+
+```yaml
+mixins:
+- docker
+```
+
+## Mixin Commands
+The commands available are docker pull, push, build, run, remove, and login. 
+
+## Mixin Syntax
+The same syntax applies for install, upgrade, and uninstall.
```

Comment:
For all of the examples below, it would help to provide yaml values for the commands so that people get an idea of what are acceptable values. See https://porter.sh/mixins/gcloud/#mixin-syntax for what that looks like:

```yaml
gcloud:
  description: "Description of the command"
  groups: GROUP
  command: COMMAND
  arguments:
  - arg1
  - arg2
  flags:
    FLAGNAME: FLAGVALUE
    REPEATED_FLAG:
    - FLAGVALUE1
    - FLAGVALUE2
  suppress-output: false
  outputs:
    - name: NAME
      jsonPath: JSONPATH
```

It should list every possible yaml field that could be specified as well, including arguments and flags. You are using yaml comments below to indicate optionality which is great. 👍